pkg/scheduler/apis/config/types.go

```diff
@@ -35,6 +35,12 @@ const (
 
 	// SchedulerDefaultProviderName defines the default provider names
 	SchedulerDefaultProviderName = "DefaultProvider"
+
+	// MaxCustomPriorityMaxScore is the max score UtilizationShapePoint expects.
+	MaxCustomPriorityMaxScore int64 = 10
```

Comment:
s/MaxCustomPriorityMaxScore/MaxCustomPriorityScore

Reply:
also, why not place this and MaxWeight into the section below with DefaultPercentageOfNodesToScore? we don't want any constants to be part of the exported API.

Reply:
Done.